### ISSUE TITLE
ATAC-seq single end?

### ISSUE THREAD

**Issue by lindsayhrlee** [haowenz/chromap#152]
Hi,

I understand that the default mapping option for the ATAC-seq data is pair ended, but is there a mapping option for a single ended ATAC-seq data? 

Thank you,
Lindsay

**Comment by mourisl** [comment]
Yes, you can just provide the "-1" option for single-end data.

**Comment by lindsayhrlee** [comment]
Thank you for your response! 

Is SAM file as an output also available for ATAC-seq settings? or is this only limited to Hi-C?

**Comment by mourisl** [comment]
Yes, the SAM format is supported for all the sequencing platforms.
